firmware/quadruna/RSM/src/io/io_coolant.c

```diff
@@ -103,16 +106,22 @@ bool pressure_ocsc(float v)
 
 float io_coolant_getPressureA(void)
 {
-    const float v_read = hw_adc_getVoltage(ADC1_IN12_COOLANT_PRESSURE_1);
-    app_canTx_RSM_Warning_CoolantPressureAOCSC_set(pressure_ocsc(v_read));
-    const float press_a = (v_read - PRESSURE_VOLTAGE_MIN) * PSI_PER_VOLT + MIN_PRESSURE_PSI;
-    return press_a;
+    return CLAMP(VOLTAGE_PRESSURE_CONVERSION(ADC1_IN12_COOLANT_PRESSURE_1), PRESSURE_VOLTAGE_MIN, PRESSURE_PSI_MAX);
```

Comment:
if you clamp ,make sure that there is visibility that the value has left the range, especially on a value that we broadcast.